DashAI/front/src/components/experiments/NewExperimentModal.jsx

```diff
@@ -43,7 +43,13 @@ const defaultNewExp = {
   task_name: "",
   input_columns: [],
   output_columns: [],
-  splits: { training: 0, validation: 0, testing: 0 },
+  splits: {
+    has_changed: false,
+    is_random: true,
+    train: 0.7,
+    validation: 0.1,
+    test: 0.2,
```

Comment:
If these are the default values, I think this ratio would be better: 

```suggestion
    train: 0.6,
    validation: 0.2,
    test: 0.2,
```